.gitmodules

```diff
@@ -0,0 +1,6 @@
+[submodule "docs/k8ssandra"]
```

Comment:
We don't need submodules anymore now that Docsy installed through a Hugo module.
Could you remove this file?

Reply:
done